src/api/app/models/role.rb

```diff
@@ -22,10 +22,7 @@ class NotFound < APIError
                                   message: 'is the name of an already existing role' }
 
   belongs_to :groups_roles, optional: true
-  belongs_to :attrib_type_modifiable_bies, class_name: 'AttribTypeModifiableBy', optional: true
```

Comment:
Isn't this used [here](https://github.com/openSUSE/open-build-service/blob/27766efc40396e883bba1a4300a15ed36e5b1774/src/api/app/models/attrib_type.rb#L130)?

Reply:
Nop. That one uses the `attrib_type_modifiable_bies` association from the `AttribType` model, defined [here](https://github.com/openSUSE/open-build-service/blob/27766efc40396e883bba1a4300a15ed36e5b1774/src/api/app/models/attrib_type.rb#L20).